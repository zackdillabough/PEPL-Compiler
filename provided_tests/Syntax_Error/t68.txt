POLY A(R) = R + 2 R + 1;
POLY F(Q, W, E, R, T, Y) = Q^121 + W - R^2 - 1;
POLY B(R) = R + 2 R + 1;
START
A(3);
INPUT X;
B(X);
INPUT Y;
INPUT Z;
F(0, Y, X, X, Z, Z);
9 8 6 1 3 4
