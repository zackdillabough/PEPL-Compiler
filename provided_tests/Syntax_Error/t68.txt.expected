10
28
-74
